src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -744,6 +753,86 @@ def load_ldlc_medication_effect(key: str, location: str) -> pd.DataFrame:
     return df
 
 
+def load_relative_risk_categorical_sbp(key: str, location: str) -> pd.DataFrame:
+    distributions = data_values.RELATIVE_RISK_SBP_ON_HEART_FAILURE_DISTRIBUTIONS
+    population_structure = load_population_structure(
+        data_keys.POPULATION.STRUCTURE, location
+    ).droplevel("location")
+
+    # define TMREL data
+    baseline_hf_rrs = pd.DataFrame(
+        1.0, index=population_structure.index, columns=ARTIFACT_COLUMNS
+    )
+    baseline_hf_rrs["parameter"] = "cat4"
+
+    # define exposed groups data
+    exposed_groups_rrs = []
+    for sbp_category, distribution in distributions:
+        rr_data = get_random_variable_draws(DRAW_COUNT, (sbp_category, distribution))
+        # relative risks of 1 for ages without heart failure (under 25)
+        under_25_data = pd.DataFrame(
+            1,
+            index=population_structure.query("age_start<=25").index,
+            columns=ARTIFACT_COLUMNS,
+        )
+
+        over_25_values = np.repeat(
+            [rr_data], len(population_structure.query("age_start>25")), axis=0
+        )
+        over_25_data = pd.DataFrame(
+            over_25_values,
+            index=population_structure.query("age_start>25").index,
+            columns=ARTIFACT_COLUMNS,
+        )
+
+        relative_risk_heart_failure = pd.concat([under_25_data, over_25_data])
+        relative_risk_heart_failure["parameter"] = sbp_category
+
+        exposed_groups_rrs.append(relative_risk_heart_failure)
+
+    exposed_rrs = pd.concat(exposed_groups_rrs)
+
+    # define all heart failure data
+    heart_failure_rrs = pd.concat([baseline_hf_rrs, exposed_rrs])
+    heart_failure_rrs["affected_entity"] = "heart_failure"
+    heart_failure_rrs["affected_measure"] = "incidence_rate"
+    heart_failure_rrs = heart_failure_rrs.set_index(
+        ["affected_entity", "affected_measure", "parameter"], append=True
+    )
+
+    return heart_failure_rrs
```

Comment:
You should sort the index so that the ordering matches the other RRs in the artifact. It doesn't really matter, but it makes inspection of the data easier.